src/api/ExperimentIdLink.js

```diff
@@ -0,0 +1,57 @@
+import { setContext } from 'apollo-link-context';
+import _set from 'lodash/set';
+import cookieStore from '@/util/cookieStore';
+
+// Experiment Ids from setting that will be passed in the X-Experiment Header
+const targetIds = [
+	'category_row_hillclimb',
+	'redirect_to_login',
+	'ml_loan_to_loan',
+];
+
+function buildExpHeaders() {
+	// kiva specific experiment cookie
+	const exps = cookieStore.get('uiab');
+	// handle missing cookie
+	if (typeof exps === 'undefined') {
+		return '';
+	}
+	const expCookieArray = exps.split('|');
+	// filter experiments to only those containing targeted ids
+	const targetExps = expCookieArray.filter(exp => {
+		let matched = false;
+		targetIds.forEach(expId => {
+			if (exp.indexOf(expId) !== -1) {
+				matched = true;
+				return true;
+			}
+		});
+		return matched;
+	});
+
+	// set experiment id:version then concatenate
+	let experimentHeader = '';
+	targetExps.forEach((target, index) => {
+		const expSegments = target.split(':');
+		if (expSegments.length && expSegments.length > 1) {
+			experimentHeader += `${expSegments[0]}:${expSegments[1]}`;
```

Comment:
I mobile is using a semi-colon to separate the id and group [link](https://kiva.atlassian.net/wiki/spaces/DS/pages/556236911/Experiment+Headers+in+GraphQL) and that is what we've coded the [ml service](https://github.com/kiva/kiva-ml-service/blob/master/src/main/kotlin/org/kiva/ml/entities/Experiment.kt#L3) to handle - would it be a problem to have this separate by `;`?

Reply:
A little peculiar in the JS world but sure I can update.